extensions/content-health-monitor/content-health-monitor.qmd

```diff
@@ -36,19 +36,37 @@ def get_env_var(var_name, description=""):
     value = os.environ.get(var_name, "")
     if not value:
         show_instructions = True
-        instruction = f"Please set the <b>{var_name}</b> environment variable and re-render the report."
+        
+        # Generic instruction for most variables
+        if var_name != "MONITORED_CONTENT_GUID":
+            instruction = f"Please set the <code>{var_name}</code> environment variable."
+        # Detailed instructions for MONITORED_CONTENT_GUID
+        else:
+            instruction = (
+                f"Open the <b>Content Settings</b> panel, then navigate to the <b>Vars</b> pane. "
+                f"Create a new environment variable named <code>{var_name}</code>, set its value to the GUID of the content you want to monitor, "
+                f"and click <b>Add Variable</b>. When you're done, click <b>Refresh Report</b> in the upper right toolbar menu to continue."
```

Comment:
Just leaving a comment to banish the intrusive thought: It's fine to assume that if they've gotten far enough to publish the Content Health Monitor, they know what a GUID is and where to find it?